setup.cfg

```diff
@@ -0,0 +1,20 @@
+[metadata]
+name = nymph
+version = 2.0.0
```

Comment:
I suggest to use this https://github.com/pypa/setuptools_scm instead. That way the version is always in sync with the git version tag.

Reply:
Fine with me to merge without this.